src/components/apps/admin/referenceGenomes/TableView.js

```diff
@@ -0,0 +1,193 @@
+        if (selectedFlatRows.length > 0) {
+            console.log(
+                "Selected => " + JSON.stringify(selectedFlatRows[0].original)
+            );
+        }
+    }, [selectedFlatRows]);
+
+    console.log("No.of rows selected: " + Object.keys(selectedRowIds).length); */
+    // Render the UI for your table
+    return (
+        <>
+            <TableToolbar
+                baseId={tableId}
+                preGlobalFilteredRows={preGlobalFilteredRows}
+                setGlobalFilter={setGlobalFilter}
+                globalFilter={globalFilter}
+                onAddClicked={onAddClicked}
+                onDeletedClicked={onDeletedClicked}
+            />
+            <TableContainer component={Paper} style={{ overflow: "auto" }}>
+                <MaUTable size="small" stickyHeader {...getTableProps()}>
+                    <TableHead>
+                        {headerGroups.map((headerGroup) => (
+                            <TableRow {...headerGroup.getHeaderGroupProps()}>
+                                {headerGroup.headers.map((column) => (
+                                    <TableCell
+                                        {...(column.id === "selection"
+                                            ? column.getHeaderProps()
+                                            : column.getHeaderProps(
+                                                  column.getSortByToggleProps()
+                                              ))}
+                                    >
+                                        {column.render("Header")}
+                                        {column.id !== "selection" ? (
+                                            <TableSortLabel
+                                                active={column.isSorted}
+                                                direction={
+                                                    column.isSortedDesc
+                                                        ? "desc"
+                                                        : "asc"
+                                                }
+                                            />
+                                        ) : null}
+                                    </TableCell>
+                                ))}
+                            </TableRow>
+                        ))}
+                    </TableHead>
+                    <TableBody>
+                        {rows.map((row, i) => {
+                            prepareRow(row);
+                            return (
+                                <TableRow
+                                    {...row.getRowProps()}
+                                    onClick={() => {
+                                        toggleRowSelected(
+                                            row.index,
+                                            !row.isSelected
+                                        );
+                                    }}
```

Comment:
Logic to select a row on clicking anywhere on the row. By default selection happens only when clicked on checkbox.